### DISCUSSION TITLE
Return-Path and Resent-Sender header

### DISCUSSION THREAD

**Discussion by jshapiro2** [postalserver/postal#2771] [Development discussions]
Hello,

Where is the hdmkak value coming from in the below and what is the purpose?  Is this configurable?
![image](https://github.com/postalserver/postal/assets/5223052/7c53f19f-588a-4243-bb11-a58173ba2421)

**Comment by Pacerino** [comment]
https://datatracker.ietf.org/doc/html/rfc4021#section-2.1.16
https://datatracker.ietf.org/doc/html/rfc4021#section-2.1.22

This is an address for the receiving where to send bounce/debug messages. The address is random generated from the postal installation.

hello@world.org sends an mail to me@here.com. The message can't, for which reason ever, be delivered to the inbox of me@here.com. The Mailserver from here.com sends a bounce message to the address in the Return-Path. In your case `hdmkak@psrp...`. Postal gets this message, shows it to you in the "Incoming Message" Tab and if it can link it to the original message.

I'm actually not sure if you can change that. Probably yes with some changes in the database.


![image](https://github.com/postalserver/postal/assets/22194063/44d2658e-affb-4640-828b-1f75d3ff99ab)
